server/routes/issues.js

```diff
@@ -5,9 +5,10 @@ const controller = require('../controllers/issues');
 
 const router = express.Router();
 
-router.get('/:boardId', controller.getIssuesByBoardId);
-router.put('/:id', controller.editIssue);
+router.get('/:columnId', controller.getIssuesByColumnId);
+router.get('/:issueId', controller.getIssue);
+router.put('/:issueId', controller.editIssue);
 router.post('/', controller.createIssue);
-router.delete('/:id', controller.removeIssue);
+router.delete('/:issueId', controller.removeIssue);
```

Comment:
Common remarks:
#### Change readme.md file. Add guide how to run the project. Like 
1. install mongo db,
2. configure smth
3. npm install
4. npm run start
#### Edit package.json file, remove unnecessary lines.
#### Add eslint.
#### Change formatting in all the files: more spaces (after function args, () => {}, { key1: value1, key2: value2 }